src/CMakeLists.txt

```diff
@@ -130,10 +103,6 @@ set (NN_SOURCES
     utils/sleep.c
     utils/thread.h
     utils/thread.c
-    utils/thread_posix.h
```

Comment:
I see. That said, the number of source files touched in this commit is strictly limited to what's required based on what were discrepancies (bugs) in the previous autotools and CMake build, and changes this far don't really get into the type of refactoring we're talking about. I don't know if it would explicitly break from simply removing these changes from the commit, but it doesn't seem worth it to remove it.